spec/spec_helper.rb

```diff
@@ -76,4 +76,98 @@ def self.drop_objects
       ActiveRecord::Base.connection.execute("DROP TYPE #{enum} CASCADE")
     end
   end
+
+  TableLock = Struct.new(:table, :lock_type, :granted, :pid)
+  def self.locks_for_table(table, connection:)
```

Comment:
Is it worth putting `DatabaseHelper` in a different file at this point?

Reply:
I was wondering that; I was also wondering if it should be split between "used by tests" and "used by spec_helper.rb".

Thoughts?

Reply:
I like the idea of splitting them up. I don't think the "used by spec helper" methods necessarily need to be in a different file though. Might make sense to inline those, but I don't feel too strongly

Reply:
Put the new stuff in a new file.